bin/utils/process-definition.sh

```diff
@@ -17,6 +17,6 @@ fi
 docker run --rm --name json2xlsx \
   -v ${definition_input_dir}:/tmp/ccd-definition \
   -v ${definition_output_file}:/tmp/ccd-definition.xlsx \
-  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://docker.for.mac.localhost:4000} \
+  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://${DOCKERFORLINUX}:4000} \
```

Comment:
Make sure not to commit this, as it will break master! Only for local testing on linux vm

Reply:
can you check and resolve please